core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -455,7 +455,8 @@ public String getPhysicalDestinationShortName(boolean throwException) throws Jms
 			if (throwException) {
 				throw new JmsException(e);
 			}
-			log.warn("[" + getName() + "] got exception in getPhysicalDestinationShortName", e);
+			// Do not write stacktrace to avoid cluttering the log, see https://github.com/ibissource/iaf/issues/4381
+			log.debug("[{}] got exception in getPhysicalDestinationShortName: {}", this::getName, e::getMessage);
```

Comment:
Is het niet mooier om te kijken of getDestinationName gevuld is en anders null terug te geven?